tests/abi/tools/revng-check-compatibility-with-abi/ABIRuntimeTestResultParser.cpp

```diff
@@ -11,25 +11,101 @@
 
 #include "ABIRuntimeTestResultParser.h"
 
-namespace State {
+namespace abi::runtime_test {
+
+struct RawState {
+  std::vector<RawWord> Registers;
+  std::vector<std::byte> Stack;
+
+  bool empty() const { return Registers.empty() && Stack.empty(); }
+
+  State extract(model::Architecture::Values Architecture) && {
+    SortedVector<Register> ResultRegisters;
```

Comment:
Do not use `SortedVector`. Use `std::set`. `batch_insert` is fragile and confusing and this is not performance critical code.

Do not treat testing code as performance critical code. The readability of this file is seriously compromised by the attempts to make it performant.

Reply:
`SortedVector` is gone.